src/TreeMap/App.svelte

```diff
@@ -80,12 +80,28 @@
 
         map.on("load", () => {
             fetchContributionsData.then((contributions) => {
+                let unique_coordinates = [];
+                let unique_coordinates_count = [];
+                contributions = Object.values(contributions).filter((c) => {
+                  if (c.geometry && c.geometry.type === 'Point' && c.geometry.coordinates) {
+                    if (unique_coordinates_count[c.geometry.coordinates.toString()]) {
+                      unique_coordinates_count[c.geometry.coordinates.toString()] += Number(c.properties.treeCount);
```

Comment:
Collect the sum of the treeCount values for unique coordinates in this associative array.